scripts/docker/Dockerfile.hipcc

```diff
@@ -0,0 +1,29 @@
+ARG BASE=rocm/dev-ubuntu-18.04:3.0
+FROM $BASE
+
+RUN apt-get update && apt-get install -y \
+        kmod \
+        wget \
+        ccache \
+        && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+ENV PATH=/opt/rocm/bin:$PATH
+
+ARG CMAKE_VERSION=3.10.3
```

Comment:
Any reason why you take an older version?

Reply:
Kokkos supports 3.10 but we do have other builds with that version so I can bump it if you like.

Reply:
its fine for now.